project/AppDependencies.scala

```diff
@@ -30,7 +30,6 @@ object AppDependencies {
   )
 
   val itDependencies: Seq[ModuleID] = Seq(
-    "org.wiremock" % "wiremock" % wiremockVersion % Test
```

Comment:
wiremockVersion val left in - not used
PR Bot comments not addressed

Reply:
val removed and dependency